_includes/v23.1/cdc/pubsub-performance-setting.md

```diff
@@ -0,0 +1,3 @@
+{{site.data.alerts.callout_info}}
+{% include_cached new-in.html version="v23.1" %} Enable the `changefeed.new_pubsub_sink_enabled` [cluster setting](cluster-settings.html) to improve the throughput of changefeeds emitting to Pub/Sub sinks.
```

Comment:
Would it make sense to link to `changefeed-sinks.html#google-cloud-pub-sub` here?

Reply:
This is a good point! I haven't mainly because the areas where this sits is right by sentences referring to the changefeed sinks page. (Or it sits on the Changefeed Sinks page). But, I might as well add an if/else block in case a reader doesn't see the Changefeed Sinks link.